{
  assertThrows(log,new Callable<Object>(){
    @Override public Object call() throws Exception {
      return data.getAsync(null);
    }
  }
,NullPointerException.class,"Ouch! Argument cannot be null: key");
}
